### PR TITLE
Add French localization of login page

### PR DESCRIPTION
**Pull request by SoleneGK**
Completed french localization of login page, removed unused labels
Sans titre

### CODE REVIEW

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2019, Damian Szczepanik
```
Comment (aHenryJard) [inline core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties:3]:
```suggestion
# Copyright (c) 2020
```

File: core/src/main/resources/jenkins/model/Jenkins/login_fr.properties [hunk 2]
```diff
@@ -20,8 +20,9 @@
...
-User=Utilisateur
+signIn=S''identifier
+signUp=<a href="signup">Cr\u00e9ez un compte</a> si vous n''\u00eates pas encore membre.
+Invalid\ username\ or\ password=Nom d''utilisateur ou mot de passe incorrect
+Username=Utilisateur
 Password=Mot de passe
-Remember\ me\ on\ this\ computer=Conserver mes informations sur cet ordinateur
-login=S''identifier
-signUp=<a href="signup">Cr�ez un compte</a> si vous n''�tes pas encore membre.
+Keep\ me\ signed\ in=Conserver mes informations sur cet ordinateur
```
Comment (PierreBtz) [inline core/src/main/resources/jenkins/model/Jenkins/login_fr.properties:28]: J'ai un doute sur celle là, peut être plutôt "Rester connecter", "Garder ma session ouverte"?

Comment (SoleneGK) [reply core/src/main/resources/jenkins/model/Jenkins/login_fr.properties:28]: J'avais un doute aussi mais je ne trouvais pas de formulation non genrée. Je vais prendre la 2e proposition, merci :)

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Welcome\ to\ Jenkins!=Bienvenue dans Jenkins !
```
Comment (alecharp) [inline core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties:23]:
```suggestion
Welcome\ to\ Jenkins!=Bienvenue dans Jenkins !

```

File: core/src/main/resources/jenkins/model/Jenkins/login_fr.properties [hunk 4]
```diff
@@ -20,8 +20,9 @@
...
-User=Utilisateur
+signIn=S''identifier
+signUp=<a href="signup">Cr\u00e9ez un compte</a> si vous n''\u00eates pas encore membre.
+Invalid\ username\ or\ password=Nom d''utilisateur ou mot de passe incorrect
+Username=Utilisateur
 Password=Mot de passe
-Remember\ me\ on\ this\ computer=Conserver mes informations sur cet ordinateur
-login=S''identifier
-signUp=<a href="signup">Cr�ez un compte</a> si vous n''�tes pas encore membre.
+Keep\ me\ signed\ in=Garder ma session ouverte
```
Comment (alecharp) [inline core/src/main/resources/jenkins/model/Jenkins/login_fr.properties:28]:
```suggestion
Keep\ me\ signed\ in=Garder ma session ouverte

```

File: core/src/main/resources/jenkins/model/Jenkins/login_fr.properties [hunk 5]
```diff
@@ -20,8 +20,9 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-User=Utilisateur
+signIn=S''identifier
+signUp=<a href="signup">Cr\u00e9ez un compte</a> si vous n''\u00eates pas encore membre.
+Invalid\ username\ or\ password=Nom d''utilisateur ou mot de passe incorrect
+Username=Utilisateur
```
Comment (alecharp) [inline core/src/main/resources/jenkins/model/Jenkins/login_fr.properties:26]: This even means that the current login page in french is not showing `Utilisateur` but `Username`.. so Thanks!

**Review by alecharp — APPROVED** [review]
Looks good. Small suggestions but not blockers.

**Review by aHenryJard — APPROVED** [review]
Looks good to me.

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2020
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties:3]: Copyright needs to mention the copyright holders. If you are not comfortable with using your name here, consider adding "Jenkins contributors"

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me, just a minor comment about the license.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

File: core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties (hunk 6 above)
Comment (SoleneGK) [reply core/src/main/resources/jenkins/model/DefaultSimplePageDecorator/simple-header_fr.properties]: Thank you, I have added my name :)

**Comment by oleg-nenashev** [comment]
Thanks for your contribution! If this is a Hacktoberfest pull request, please let us know so that we can label it properly and facilitate reviews. Check out [this page](https://www.jenkins.io/events/hacktoberfest/) for more information about Jenkins in Hacktoberfest: guidelines, featured projects, etc. 

[image](https://www.jenkins.io/events/hacktoberfest/)

**Pull request merged**